docs/advanced-guide/handling-file/page.md

```diff
@@ -46,6 +46,48 @@ To create subdirectories as well
 err := ctx.File.MkdirAll("my_dir/sub_dir", os.ModePerm)
 ```
 
+### Get current Directory
+
+```go
+currentDir, err := ctx.File.Getwd()
+```
+
+### Change current Directory
+
+To switch to parent directory
+```go
+currentDir, err := ctx.File.Chdir("..")
+```
+
+To switch to another directory in same parent directory
+```go
+currentDir, err := ctx.File.Chdir("../my_dir2")
+```
+
+To switch to child directory
```

Comment:
What about this?

```suggestion
To switch to a subfolder of the current directory
```

`child` could let think there is a kind of magic that detects the first child or something